src/scripts/main.js

```diff
@@ -1 +1,74 @@
 'use strict';
+
+const showNotification = (message, type) => {
+  const messageEl = document.createElement('div');
+
+  messageEl.className = type;
+  messageEl.setAttribute('data-qa', 'notification');
+  messageEl.textContent = message;
+  document.body.append(messageEl);
+};
+
+const firstPromise = new Promise((resolve, reject) => {
+  const handleClick = (e) => {
+    if (e.button === 0) {
+      document.removeEventListener('mousedown', handleClick);
+      resolve('First promise was resolved');
+    }
+  };
+
+  document.addEventListener('mousedown', handleClick);
+
+  setTimeout(() => {
+    document.removeEventListener('mousedown', handleClick);
+    reject(new Error('First promise was rejected'));
+  }, 3000);
```

Comment:
The timeout for rejecting the first promise is set to 3000 milliseconds (3 seconds). Ensure this is the intended behavior, as it might not provide enough time for the user to click.